src/IdeHelperServiceProvider.php

```diff
@@ -32,6 +35,13 @@ class IdeHelperServiceProvider extends ServiceProvider implements DeferrableProv
      */
     public function boot()
     {
+        if ($this->app['config']->get('ide-helper.post_migrate', false)) {
```

Comment:
Thanks for the other change and the clarification! I think for consistency, we also should default to `[]` here now:
```suggestion
        if ($this->app['config']->get('ide-helper.post_migrate', [])) {
```

Reply:
I thought leaving this as false since its a condition but ok, changed it.